src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,77 @@
+package duke;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * deals with loading and saving tasks in the file
+ */
+public class Storage {
+    private String directory;
+    private String filePath;
+
+    public Storage(String directory, String filePath) {
+        this.directory = directory;
+        this.filePath = filePath;
+    }
+
+    /**
+     * rewrite the file with newly edited tasks after the user quits the app
+     * @param list_of_inputs
+     * @throws IOException
+     */
+    public void reSavingFiles (ArrayList<Task> list_of_inputs) throws IOException {
```

Comment:
Just to add on to @weijuey, resaving seems to be a correct word, so maybe you can consider `resavingFiles`?